### ISSUE TITLE
Cannot limit the training data to a specific camera distance

### ISSUE THREAD

**Issue by siebenkopf** [bioidiap/bob.db.scface#2]
I wanted to perform an experiment where I limit the training data to match the data from the test set. Unfortunately, currently there is no way to do that by simply querying the database.

Note that the training set should contain images from all cameras, including the "frontal" that are usually used for enrollment.
